### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by Bornittah**
### Project requirements
-  set up controllers.

-  Create the following for each URL 
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files should include only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user

-  Add code to handle **only** the 4 required URLs.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```
Comment (mikethreels) [inline app/controllers/posts_controller.rb]: - [ ] for this controller the index and the post actions should have been created.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```
Comment (mikethreels) [inline app/controllers/users_controller.rb]: - [ ] for this controller the index and the post actions should have been created.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Bornittah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Bornittah 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Created necessary actions in `Users` and `Posts` controllers ✔️
- Created routes for the actions in the `routes.rb` file ✔️  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**